xls/contrib/mlir/IR/xls_ops.cc

```diff
@@ -390,6 +390,32 @@ Type getI1TypeOf(Type type) {
 
 namespace mlir::xls {
 
+LogicalResult LiteralOp::verifyRegions() {
+  Block& b = getInitializerBlock();
+  // Verify return op & typing:
+  auto ret = dyn_cast<YieldOp>(b.getTerminator());
+  if (!ret) {
```

Comment:
Shouldn't this hold due to the verification already generated? (e.g., `SingleBlockImplicitTerminator<"YieldOp">`).